### ISSUE TITLE
IDEA: simplify comparison

### ISSUE THREAD

**Issue by darribas** [alan-turing-institute/demoland-web#18]
I wonder if the current interface for comparisons ("Compare with") is over complicated. Right now, it gives the impression  you can compare any pair of scenarios. But the difference maps are only available for scenario vs baseline pairs. I wonder if we could simplify this interface to only give the option of comparing a scenario with the baseline (so the user doesn't need to select anything). What'd you think @yongrenjie ?

**Comment by martinfleis** [comment]
I thought we can do difference between scenarios, not only between a scenario and a a baseline. If we can't right now, it would be nice if we could. I find it useful.

**Comment by yongrenjie** [comment]
Yeah, you can get the difference between any two scenarios for any given indicator.

(However, I took the 'difference' option out when visualising land use because I didn't really see how to provide a diff for land use.)
